pipelines/tests/unit/extractor/test_bigquery_metadata_extractor.py

```diff
@@ -430,10 +587,14 @@ def test_keypath_and_pagesize_can_be_set(self, mock_build):
             )
 
     @patch("whale.extractor.base_bigquery_extractor.build")
-    def test_table_part_of_table_date_range(self, mock_build):
-        mock_build.return_value = MockBigQueryClient(
+    @patch("whale.extractor.base_bigquery_extractor.datacatalog_v1")
+    def test_table_part_of_table_date_range(self, mock_datacatalogue, mock_bigquery):
```

Comment:
BRITISH ENGLISH!? UNACCEPTABLE!
Just kidding, please don't change it. :)

Reply:
Oh haha my bad, I mixed it up 😂  [Google style...](https://github.com/googleapis/python-datacatalog/)